lib/streamlit/__init__.py

```diff
@@ -230,74 +231,100 @@ def set_option(key, value):
 
 def write(*args, **kwargs):
     """Write arguments to the app.
-    This is the swiss-army knife of Streamlit commands. It does different
-    things depending on what you throw at it.
+
+    This is the swiss-army knife of Streamlit commands. It does different things
```

Comment:
Here is the most pedantic, obnoxious review comment ever:

I think this should be "Swiss Army knife" instead of "swiss-army knife" (the Swiss Army bit is a proper noun and not hyphenated).

Reply:
Agreed, didn't have my pedantic glasses on yesterday

Reply:
Can leave as-is, I'm about to touch that file anyway and can fix it